includes/subscriptions.php

```diff
@@ -0,0 +1,252 @@
+<?php
+
+class Migrator_CLI_Subscriptions {
+	public function import( $assoc_args ) {
+
+		try {
+			Migrator_CLI_Utils::health_check();
+
+			if ( !isset( $assoc_args['subscriptions_export_file'] ) || !isset( $assoc_args['orders_export_file'] ) ) {
+				WP_CLI::line( WP_CLI::colorize( '%RExport Files not provided. Go to Skio dashboard > Export and export both subscriptions and orders. Then pass the file path to --subscriptions_export_file and --orders_export_file args' ) );
+				return;
+			}
+
+			Migrator_CLI_Utils::disable_sequential_orders();
+
+			$skio_orders = $this->get_data_from_file( $assoc_args['orders_export_file'] );
+
+			WP_CLI::line( 'Adding Subscription ids to orders' );
+			$this->add_subscription_id_to_orders( $skio_orders );
+
+			$skio_subscriptions = $this->get_data_from_file( $assoc_args['subscriptions_export_file'] );
+
+			WP_CLI::line( 'Creating Subscriptions' );
+			$this->create_or_update_subscriptions( $skio_subscriptions );
+
+			Migrator_CLI_Utils::enable_sequential_orders();
+		} catch ( \Exception $e ) {
+			WP_CLI::line( WP_CLI::colorize( ' %rError: ' . $e->getMessage() ) );
+		}
+
+		WP_CLI::line( WP_CLI::colorize( '%GDone%n' ) );
+	}
+
+	// Clones the line item and adds it to the subscription.
+	private function clone_item_to_subscription( $item, $subscription ) {
+		$new_item = clone $item;
+		$new_item->set_id( 0 );
+		$new_item->set_order_id( $subscription->get_id() );
+		$new_item->save();
+
+		$subscription->add_item( $new_item );
+	}
+
+	private function get_data_from_file( $file )
```

Comment:
```suggestion
	private function get_data_from_file( $file )
	{
		if ( ! file_exists( $file ) ) {
			throw new \Exception( 'File not found: ' . $file );
		}
		return json_decode(file_get_contents( $file ), true);
	}
```

Result: 

<img width="700" alt="image" src="https://github.com/woocommerce/migrator-cli/assets/4162931/9f7eebbb-639f-40e7-8578-e4107c78f4ef">